org.baedert.liblistbox.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id" : "org.baedert.liblistbox",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "demo",
+    "finish-args" : [
+        "--socket=wayland",
+        "--share=network"
+    ],
+    "build-options" : {
+        "cflags" : "-g",
+        "env" : {
+            "GTK_INTERACTIVE" : "true"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "wayland",
+            "buildsystem" : "autotools",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/wayland-project/wayland.git",
+                    "tag" : "1.15.0"
+                }
+            ],
+            "config-opts" : [
+                "--disable-documentation"
+            ]
+        },
+        {
+            "name" : "graphene",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ebassi/graphene.git",
+                    "tag" : "1.8.2"
+                }
+            ],
+            "config-opts" : [
+                "--libdir=lib",
+                "-Dintrospection=false",
+                "-Dtests=false",
+                "-Dbenchmarks=false"
+            ]
+        },
+        {
+            "name" : "gtk-4.0",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+                "--prefix=/app",
+                "--libdir=lib",
```

Comment:
Aren't `--libdir` and `--prefix` supplied by default with the proper values by flatpak(-builder) automatically?